docs/advanced/errors.md

```diff
@@ -2,6 +2,8 @@
 
 NGXS uses Angular's default `ErrorHandler` class, so if an action throws an error, Angular's `ErrorHandler` is called. You can easily override this flow by providing your own handler like so:
 
+NGXS propagates unhandled errors inside `@Action` to Angular's `ErrorHandler`. You can provide your own implementation of `ErrorHandler` and handle it the way you need like this:
```

Comment:
@joaqcid I just merged and then saw this. Was this supposed to replace the previous paragraph?
You can do another PR if you need to fix it.

Reply:
actually it was just a different option, just add it to see what you thought

Reply:
i didnt mean to include it in the PR, i forgot to unstage it, let me know if you want me to revert it @markwhitfeld